### DISCUSSION TITLE
is it possible to install Raspberry Pi OS with crouton?

### DISCUSSION THREAD

**Discussion by curlyhairedgurl** [dnschneid/crouton#4699] [Q&A]
Is it possible to install Raspberry Pi OS (Raspbian) by using crouton? I would like to use the OS on my chromebook instead of having to connect my RPi to a monitor.

Thank you for reading.

**Comment by TomTravis** [comment]
in change root yes  a PIE CR can exist the HW is not the same RPI can do chroot alone also
it not that difficult [chroot](https://wiki.debian.org/chroot) RPI is nice it boots pid 1

**Comment by TomTravis** [comment]
it(rpi) could be a remote X server no problem ssh to start the session [Xserver](http://www.multi-seat.com/x2go/)

i just bought new HW to do the exact same thing Lenovo Intel I7 box 16TB e-sata 1000mb/sec  SSD drive aliexpress  32gb memory SUSE server used charity support(purchased), i also have [Ryzen](https://www.amd.com/en/products/cpu/amd-ryzen-9-5900x) box coming to see. many Kernel to test

**Comment by ghost** [comment]
Discussion #4393 was about this, although my answer was specifically about desktop environments, if you are interested get Raspberry Pi OS to run on a Chromebook look into it, and Crouton's scripts. Good luck!

**Comment by TomTravis** [comment]
just use Debootstrap no need for crouton in Rpi and use have new chroot
Myself just
apt install x2goserver* on the rpi
then
apt install x2goclient* on the crouton chroot and connect to the rpi ip address
![Screenshot at 2022-06-05 17-35-46](https://user-images.githubusercontent.com/46386261/172077244-23f2dfc6-3f2d-4a2b-8930-854a008cd3a6.png)

then you have access to rpi hw on your workstation
